amazon-lambda/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,39 @@
+= Amazon Lambda Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Amazon Lambda (Amazon Lambda Connector) enables you to execute AWS Lambda operations within your Mule flows. You can then use the response of the Lambda operation to process another Mule flow as needed.
+
+For compatibility information and fixed issues, see the Amazon Lambda Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The AWS Lambda API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* AWS Identity and Access Management (IAM) Access Key credentials 
+* Access to the AWS Lambda target resource
+* Anypoint Studio version 7.3 or later
+
+== Common Use Case for the Connector
+
+Data streams in from communication devices, from peripherals interacting with the physical world, or from user input devices. In most cases, this data arrives in short bursts, or a few bytes at a time, in formats that can be easily parsed. There are times, however, when your application must manage large volumes of streaming input data and moving that data to temporary storage for later processing might not be an adequate solution. Amazon Lambda Connector enables you to send the data stream to an AWS Lambda application designed to quickly pull and process the required information so that you can manage the necessary realtime tasks without slowing down your main application.
```

Comment:
Do we typically begin the use case text with a sentence fragment? 
I was a bit jarred to see this start with "Data streams in from," which initially sounded like "data" was the subject and "streams in from" the verb....

Reply:
I wonder if we could replace all of this with the final sentence alone, or modified with an intro something like "When your app manages large volumes of streaming input, Amazon Lambda Connector enables you to...."

Reply:
haha No, that's a C&P error.